eng/tools/generator/cmd/issue/issueCmd.go

```diff
@@ -196,10 +197,11 @@ func issueHasLabel(issue *github.Issue, label IssueLabel) bool {
 type IssueLabel string
 
 const (
-	GoLabel              IssueLabel = "GO"
+	GoLabel              IssueLabel = "Go"
 	AutoLinkLabel        IssueLabel = "auto-link"
 	PRreadyLabel         IssueLabel = "PRready"
 	InconsistentTagLabel IssueLabel = "Inconsistent tag"
+	TypeSpecLabel        IssueLabel = "TypeSpec"
```

Comment:
As synced offline, we shall remove dependency on the `TypeSpec` label in the release request issue.